hangman/lib/game/Game.ex

```diff
@@ -0,0 +1,145 @@
+defmodule Hangman.Game do
```

Comment:
## -2 idiomatic

The naming conventions for files and directories are 

1. all lower case

2. directories are named for the parent module, not the child module

So this file should be `lib/hangman/game.ex`